BrowserServicesKit/Sources/BrowserServicesKit/PrivacyConfig/Features/PrivacyFeature.swift

```diff
@@ -25,6 +25,7 @@ public enum PrivacyFeature: String {
     case fingerprintingTemporaryStorage
     case fingerprintingBattery
     case fingerprintingScreenSize
+    case fingerprintingCanvas
```

Comment:
Is this needed for the config serialization? Can we pull this list from the config potentially or support them dynamically?